packages/mrc-ide-covidsim/src/imperial-params.ts

```diff
@@ -110,19 +110,25 @@ function daysBetween(t0: DateTime, t1: DateTime): number {
 
 export function assignPreParameters(p: {}, input: input.ModelParameters) {
   const calibrationDate = dateFromAPI(input.calibrationDate)
+  const interventionStart =
+    input.interventionPeriods.length > 0
+      ? dateFromAPI(input.interventionPeriods[0].startDate)
+      : DateTime.utc()
```

Comment:
This is a little strange if there are no intervention periods, but I see there is a model parameter for it below that we have to set.

Reply:
From the UI, it should not be possible to create a simulation with no periods set. Though, I guess we do need to handle this because someone _could_ send a request directly to the mode-runner.

Reply:
Let's tackle that separately. My concern is that if you send an empty list directly, then it will be interpreted as "there is a single intervention starting today and extending indefinitely", but as you say this can't be done from the UI.